implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -296,7 +296,9 @@ impl<'a> Rpc<'a> {
     where
         T: Output + serde::Serialize,
     {
-        print_output(b, &self.opts.global_args.output_format)
+        let r = print_output(b, &self.opts.global_args.output_format)?;
```

Comment:
Would it make sense to introduce a `println_output` for this kind of output where we want a final newline?